internal/pkg/workspace/workspace.go

```diff
@@ -133,6 +133,29 @@ func (ws *Workspace) writeSummary(projectName string) error {
 	return ws.fsUtils.WriteFile(summaryPath, serializedWorkspaceSummary, 0644)
 }
 
+// AppExist returns if an application exists in the workspace.
+func (ws *Workspace) AppExist(appName string) (bool, error) {
```

Comment:
I feel that this function should be unexported and used internally to this package as part of the `DeleteFile` method as that's the only existing use case right now.